pytensor/graph/rewriting/basic.py

```diff
@@ -2575,8 +2575,8 @@ def print_profile(cls, stream, prof, level=0):
         for i in range(len(loop_timing)):
             loop_times = ""
             if loop_process_count[i]:
-                d = list(
-                    reversed(sorted(loop_process_count[i].items(), key=lambda a: a[1]))
+                d = sorted(
```

Comment:
I was originally skeptical about this since I thought that `sorted` returns an iterator, but checking it, it actually returns a list, so this should be fine. (On the other hand, `reversed` is an iterator.)

Reply:
Yeah it's not intuitive that they would return different objects!

Reply:
I guess you can't lazily sort stuff...